examples/checkpointing/checkpointing_example.py

```diff
@@ -0,0 +1,143 @@
+import numpy as np
+from numpy import linalg
+from math import floor
+from devito import Function, TimeFunction
+from examples.seismic import Receiver, demo_model, RickerSource
+from examples.seismic.acoustic import ForwardOperator, GradientOperator, smooth10
+from examples.checkpointing.checkpoint import DevitoCheckpoint, DevitoOperator
+from pyrevolve import Revolver
+
+
+def setup(shape, tn, spacing, time_order, space_order, nbpml):
```

Comment:
if this is seismic/acoustic based, then it should probably live in the seismic folder 

much better: move this to /seismic ; add here the most trivial example that one can come up with

Reply:
This is the most trivial (and yet useful) example I can think of, of using a devito operator with pyrevolve. "The most trivial example" lives in the examples folder of pyrevolve.